mpas_analysis/shared/mpas_xarray/mpas_xarray.py

```diff
@@ -116,11 +124,12 @@ def time_series_stat_time(timestr, daysSinceStart): #{{{
     else:
         return pd.to_timedelta(daysSinceStart.values, unit='ns')
 
-    #}}}
+    # }}}
+
 
 def preprocess_mpas(ds, onlyvars=None, selvals=None, iselvals=None,
-        timeSeriesStats=False, timestr=None,
-        yearoffset=1849, monthoffset=12, dayoffset=31): #{{{
+                    timeSeriesStats=False, timestr=None,
+                    yearoffset=1849):  # {{{
```

Comment:
We are loosing an option here in exchange of hard-coded values.  Why aren't we just setting the defaults to new values?  I think this is probably better long term by keeping the code more general.